src/js/cellProfile.js

```diff
@@ -79,7 +79,7 @@ cellProfile.prototype.populateCellProfilePopupFieldsWithData = function() {
         document.getElementById("editDisplayName").dataset.CellType = "";
     }
 
-    if (binaryCellProfileImage != null) {
+    if (binaryCellProfileImage!= null && binaryCellProfileImage!= undefined && binaryCellProfileImage != '') {
```

Comment:
It is simpler to do the following:  

    if (binaryCellProfileImage) {
        ...
    }